From 6fd693458ebbc5dd60045f8190f002d037d6625d Mon Sep 17 00:00:00 2001
From: Adam Seaton <aseaton@google.com>
Date: Mon, 13 Aug 2018 09:34:24 -0700
Subject: [PATCH 6/9] Update platform security string to 2018-10-05 on pi-dev &
 master bug: 112535135 (cherry picked from commit
 61ddf1fa27039bc83a7e4be48a35a50ab2b83c78)

Change-Id: I2f3fe055b0381640353b3ab05af8f6b80e631316
---
 core/version_defaults.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/version_defaults.mk b/core/version_defaults.mk
index 3c1c3c913..3199c09ca 100644
--- a/core/version_defaults.mk
+++ b/core/version_defaults.mk
@@ -231,7 +231,7 @@ ifndef PLATFORM_SECURITY_PATCH
     #  It must be of the form "YYYY-MM-DD" on production devices.
     #  It must match one of the Android Security Patch Level strings of the Public Security Bulletins.
     #  If there is no $PLATFORM_SECURITY_PATCH set, keep it empty.
-      PLATFORM_SECURITY_PATCH := 2018-09-05
+      PLATFORM_SECURITY_PATCH := 2018-10-05
 endif
 
 ifndef PLATFORM_SECURITY_PATCH_TIMESTAMP
-- 
2.17.1

